Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectControlled.mo

```diff
@@ -0,0 +1,299 @@
+    annotation (Line(points={{-40,200},{-10,200}}, color={0,127,255}));
+  connect(senMasFlo.m_flow, pro.u2)
+    annotation (Line(points={{-110,-269},{-110,-192},
+          {90,-192},{90,-116},{118,-116}}, color={0,0,127}));
+  connect(pro.y, cp.u)
+    annotation (Line(points={{141,-110},{180,-110}}, color={0,0,127}));
+  connect(cp.y, Q_flow)
+    annotation (Line(points={{203,-110},{320,-110}}, color={0,0,127}));
+  connect(cp.y, int.u)
+    annotation (Line(points={{203,-110},{204,-110},{204,-150},
+          {258,-150}}, color={0,0,127}));
+  connect(int.y, Q)
+    annotation (Line(points={{281,-150},{320,-150}}, color={0,0,127}));
+  connect(dTdis.y, pro.u1)
+    annotation (Line(points={{81,-104},{118,-104}}, color={0,0,127}));
+  connect(conVal.port_b, senTDisRet.port_a)
+    annotation (Line(points={{10,200},{30,200}}, color={0,127,255}));
+  connect(senTDisRet.port_b, port_bSerCoo)
+    annotation (Line(points={{50,200},{162,200},{162,-240},{262,-240},{262,-280},{300,-280}}, color={0,127,255}));
+  connect(senTDisRet.T, dTdis.u1)
+    annotation (Line(points={{40,189},{40,-98},{58,-98}}, color={0,0,127}));
+  connect(senTDisSup.T, dTdis.u2)
+    annotation (Line(points={{-170,-269},{-170,-110},{58,-110}}, color={0,0,127}));
+  connect(ope.y, cheVal.opening)
+    annotation (Line(points={{-79,-10},{-58,-10}}, color={0,0,127}));
+  connect(TSetDisRet, con.u_s)
+    annotation (Line(points={{-318,0},{-222,0}},                     color={0,0,127}));
+  connect(senTBuiRet.T, con.u_m)
+    annotation (Line(points={{-210,189},{-210,12}}, color={0,0,127}));
+  connect(jun.port_2, senTBuiSup.port_a)
+    annotation (Line(points={{-40,-280},{220,-280},{220,200},{230,200}}, color={0,127,255}));
+  connect(senTBuiSup.port_b, ports_bChiWat[1])
+    annotation (Line(points={{250,200},{300,200}}, color={0,127,255}));
+  connect(spl.port_3, cheVal.port_a)
+    annotation (Line(points={{-50,190},{-50,0},{-50,0}}, color={0,127,255}));
+  connect(cheVal.port_b, jun.port_3)
+    annotation (Line(points={{-50,-20},{-50,-270}}, color={0,127,255}));
+  connect(const.y, swi.u1)
+    annotation (Line(points={{-139,130},{-110,130},{
+          -110,80},{-82,80},{-82,78}}, color={0,0,127}));
+  connect(notCon.y, swi.u2)
+    annotation (Line(points={{-119,70},{-82,70}}, color={255,0,255}));
+  connect(con.y, swi.u3)
+    annotation (Line(points={{-199,0},{-110,0},{-110,62},
+          {-82,62}}, color={0,0,127}));
+  connect(swi.y, conVal.y)
+    annotation (Line(points={{-59,70},{0,70},{0,188}}, color={0,0,127}));
+  connect(senTBuiRet.T, onOffCon.u)
+    annotation (Line(points={{-210,189},{-210,
+          64},{-182,64}}, color={0,0,127}));
+  connect(TSetDisRet, onOffCon.reference)
+    annotation (Line(points={{-318,0},{
+          -260,0},{-260,76},{-182,76}}, color={0,0,127}));
+  connect(onOffCon.y, notCon.u)
+    annotation (Line(points={{-159,70},{-142,70}}, color={255,0,255}));
+ annotation (Placement(transformation(
```

Comment:
Is this `Placement()` annotation redundant?

Reply:
Agree, I have deleted `Placement()` part.